active/fellowship.md

```diff
@@ -2,30 +2,67 @@
 
 ## Scope of responsibilities
 
-The Fellowship Working Group manages the operation of the [Django Fellowship Program](https://www.djangoproject.com/fundraising/#fellowship-program). They select candidates to fill the fellowship, and act as line managers during the operation of the program.
+The Fellowship Working Group manages the [Django Fellowship Program](https://www.djangoproject.com/fundraising/#fellowship-program). This includes:
+
+- Interviewing candidates for open Fellowship slots and making hiring recommendations to the Board. This includes developing and maintaining interview and selection scripts, exercises, and rubrics.
+- Along with the Fellows themselves, maintaining the job description, outline of duties, and any other similar process documentation for the Fellow role (a "job manual", if you will). By and large these don't currently (as of Spring 2024), so the first group operating under this new charter will be responsible for starting to create this documentation. The Fellows and WG can maintain and update these without explicit Board approval, but should report major changes to the Board for review.
```

Comment:
Could we use a date reference that is not dependent of a given hemisphere? What about:

`(as of March/April 2024)`